### ISSUE TITLE
Travis builds hangs on Google Chrome installation 

### ISSUE THREAD

**Issue by douglascamata** [cobrateam/splinter#482]
All of splinter's builds hangs with the same error message:

```
2.39s$ sudo -E apt-get -yq --no-install-suggests --no-install-recommends --force-yes install google-chrome-stable
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:
The following packages have unmet dependencies:
 google-chrome-stable : Depends: libstdc++6 (>= 4.8.0) but 4.6.3-1ubuntu5 is to be installed
                        Depends: lsb-base (>= 4.1) but 4.0-0ubuntu20.3 is to be installed
E: Unable to correct problems, you have held broken packages.
```

**Comment by bmcculley** [comment]
I was able to get the google chrome install working, but now it's throwing an error for `travis-install.bash`.

```
$ ./travis-install.bash

if [ "${DRIVER}" = "tests/test_djangoclient.py" ]; then

    pip install -q Django==${DJANGO_VERSION}

fi

if [ "${DRIVER}" = "tests/test_webdriver_remote.py" ]; then

    export DISPLAY=:99.0

    sh -e /etc/init.d/xvfb start

    sleep 1

	wget http://goo.gl/PJUZfa -O selenium-server.jar

	java -jar selenium-server.jar > /dev/null 2>&1 &

	sleep 1

fi

Starting virtual X frame buffer: Xvfb

The command "./travis-install.bash" failed and exited with 1 during .
```

**Comment by bmcculley** [comment]
Alright, the `travis-install.bash` issue is fixed. Just needs an update for the PhantomJS tests to work.

**Comment by douglascamata** [comment]
Fixed in PR #483.
